docs/scripts_md/updateMRI_Upload.md

```diff
@@ -0,0 +1,52 @@
+# NAME
+
+updateMRI\_Upload.pl - updates database table mri\_upload according to an entry in table
+   tarchive
+
+# SYNOPSIS
+
+updateMRI\_Upload.pl \[options\] -profile prod -tarchivePath tarchivePath -source\_location source\_location
+
+- **-profile prof** : (mandatory) path (absolute or relative to the current directory) of the 
+    profile file
+- **-tarchivePath tarchivePath** : (mandatory) absolute path to the tarchive file
+- **-source\_location source\_location** : (mandatory) value to set column 
+    DecompressedLocation to for the newly created record in table mri\_upload (see below)
+- **-globLocation** : Loosen the validity check of the tarchive allowing for the 
+     possibility that the tarchive was moved to a different directory.
+- **-verbose** : Be verbose
+
+# DESCRIPTION
+
+This script first starts by reading the `prod` file (argument passed to the `-profile` switch)
+to fetch the `@db` variable, a Perl array containing four elements: the database
+name, the databse user name used to connect to the database, the password and the 
+database hostname. It then checks for an entry in table `tarchive` with the same 
+`ArchiveLocation` as the tarchive file passed on the command line. Let `T` be the 
+tarchive record found. The script will then proceed to scan table `mri_upload` for a 
+record with the same `tarchiveID` as `T`'s. If there is none (which is the expected 
```

Comment:
I am not sure how @cmadjar feels about defining it as `T`. I personally do not like it when we are describing the scripts (it makes it sound like a math proof:)), but this is just my opinion and can be ignored (i.e. not important to hold this from being merged).

Reply:
I am open to other suggestions on how to formulate this

Reply:
What's wrong with math teachers? I loved my math teachers! ;)

Is T supposed to be a TarchiveID or just a reference to the tarchive? If it is supposed to be a TarchiveID, maybe use an ID instead of T. Otherwise, T sounds fine.

Reply:
T represents the whole record (i.e the tarchive entry, with all its properties) so I guess we can keep it?

Reply:
Thanks for the clarification. That's what I thought. Looks fine the way it is unless @MounaSafiHarab disagrees.

Reply:
sounds good for me too.